Source/Core/Core/LibusbUtils.cpp

```diff
@@ -49,22 +49,22 @@ class Context::Impl
 
   libusb_context* GetContext() { return m_context; }
 
-  bool GetDeviceList(GetDeviceListCallback callback)
+  int GetDeviceList(GetDeviceListCallback callback)
```

Comment:
Am I missing something or can this function be `const`?

Reply:
If I make the function `const` I get compile errors when trying to lock on `m_device_list_mutex`.

Reply:
Ah true.  Thoughts on making that `mutable`?  I usually make my mutexes mutable in favor of const functions but I see why people would shy away from that...

Reply:
I think it could go either way, but it shouldn't be too hard to change. I don't think `const` has any benefits here, but there isn't really a downside to it either.